scripts/core/ServiceOsmApiDbHootApiDbConflate.sh

```diff
@@ -219,4 +220,15 @@ hoot is-match $HOOT_OPTS $REF_DIR/output.osm $OUTPUT_DIR/14-complete-output-Pull
 #echo ""
 #diff $REF_DIR/output.osc $OUTPUT_DIR/11b-conflated-changeset-ToBeAppliedToOsmApiDb.osc
 
+#
+echo ""
+echo "STEP 15: Cleaning up database..."
+echo ""
+if [ "$LOAD_SEC_DATA" == "true" ]; then
+  hoot delete-map -D api.db.email=$HOOT_EMAIL "$HOOT_DB_URL/5-secondary-complete-$TEST_NAME"
+fi
 
+if [ "$CONFLATE_DATA" == "true" ]; then
+  hoot delete-map -D api.db.email=$HOOT_EMAIL "$HOOT_DB_URL/8a-conflated-$TEST_NAME"
+  hoot delete-map -D api.db.email=$HOOT_EMAIL "$HOOT_DB_URL/8b-conflated-$TEST_NAME"
+fi
```

Comment:
Most of the changes are like this, just an extra call or two to `hoot delete-map` to clear out the maps that the test created.